src/io/iter_image_recordio.cc

```diff
@@ -337,6 +337,8 @@ class ImageRecordIter : public IIterator<DataInst> {
 MXNET_REGISTER_IO_ITER(ImageRecordIter_v1)
 .describe(R"code(Iterating on image RecordIO files
 
+.. note:: ``ImageRecordIter_v1`` is deprecated. Use ``ImageRecordIter`` instead.
```

Comment:
wrap the contents of the notes with extra empty line at the beginning/end otherwise it won't render on website.

Reply:
like this:
```
.. note::

   blabla...
   blabla

other doc
```

Reply:
Ok.